src/UI/Buyer/src/app/components/checkout/checkout-address/checkout-address.component.ts

```diff
@@ -114,6 +108,48 @@ export class OCMCheckoutAddress implements OnInit {
     }
   }
 
+  async handleAnonShippingAddress(newShippingAddress: Address<any>): Promise<void> {
+    if (newShippingAddress != null) {
+      this.selectedShippingAddress = newShippingAddress;
+    }
+    this.context.order.checkout.setOneTimeShippingAddress((this.selectedShippingAddress as Address))
+    this.continue.emit()
+  }
+
+  async handleLoggedInShippingAddress(newShippingAddress: Address<any>): Promise<void> {
+    if (newShippingAddress != null) {
+      this.selectedShippingAddress = await this.saveNewShippingAddress(
+        newShippingAddress
+      )
+    }
+    if (this.selectedShippingAddress) {
+      await this.context.order.checkout.setShippingAddressByID(
+        this.selectedShippingAddress
+      )
+      this.continue.emit()
+    } else {
+      // not able to create address - display suggestions to user
+      this.spinner.hide()
+    }
+  }
+
+  async SaveAddressPatchOrder(newAddress: Address<any>) {
+    if (newAddress != null) {
+      this.selectedShippingAddress = await this.saveNewShippingAddress(
+        newAddress
+      )
+    }
+    if (this.selectedShippingAddress) {
+      await this.context.order.checkout.setShippingAddressByID(
+        this.selectedShippingAddress
+      )
+      this.continue.emit()
+    } else {
+      // not able to create address - display suggestions to user
+      this.spinner.hide()
+    }
+  }
+
```

Comment:
Is there a difference between `SaveAddressPatchOrder` and `handleLoggedInShippingAddress`? If not, could we get rid of one?

Reply:
Yes we can. Not even calling the `SaveAddressPatchOrder` function so I will remove and push.